lib/datasource/remote/api/tokenmodels_repository.dart

```diff
@@ -0,0 +1,59 @@
+import 'dart:convert';
+
+import 'package:async/async.dart';
+import 'package:collection/collection.dart';
+import 'package:http/http.dart' as http;
+import 'package:seeds/datasource/remote/api/http_repo/http_repository.dart';
+import 'package:seeds/datasource/remote/api/http_repo/seeds_scopes.dart';
+import 'package:seeds/datasource/remote/model/token_model.dart';
+
+class TokenModelsRepository extends HttpRepository {
```

Comment:
please leave a comment on this Repo explaining what is doing, how its doing it and how to use it.